src/components/map/components/filter/MapFilter.tsx

```diff
@@ -27,8 +29,10 @@ export const MapFilter = ({onFilterChange}: MapFilterProps) => {
       style={style.filterButton}
       type="inline"
       compact={true}
-      interactiveColor={'interactive_2'}
+      interactiveColor="interactive_2"
       accessibilityRole="button"
+      accessibilityLabel={t(MapTexts.controls.filter.a11yLabel)}
+      accessibilityHint={t(MapTexts.controls.filter.a11yHint)}
```

Comment:
Are these necessary? Map is disabled when using screen readers.

Reply:
Very true. It was done for the zoom buttons, so I just followed the same pattern